src/main/resources/sample/examples/RC-REF/RC-REF.json

```
@@ -0,0 +1,5 @@
+{
+  "reference": {
+    "distributionID": "samuB_2607723d-507d-4cbf-bf74-12345f7064cd"
```

```suggestion
    "distributionID": "fr.health.samuB_2607723d-507d-4cbf-bf74-12345f7064cd"
```
pour être conforme avec les specs normalement